src/app.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react'
 import Form from './form.js'
+import Nav from './nav-bar.js'
+import Card from './card.js'
 
 export default class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      cards: []
+      cards: [],
+      path: ''
     }
     this.handleSubmit = this.handleSubmit.bind(this)
+    window.location.hash = '#card-list'
```

Comment:
You should let the hash be what it is and make your app respond accordingly. If I refresh the page, it should at least stay on the right view.